frontend/src/main/pages/StaffEditPage.js

```diff
@@ -0,0 +1,13 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+export default function StaffEditPage() {
+
+  // Stryker disable all : placeholder for future implementation
+  return (
+    <BasicLayout>
+      <div className="pt-2">
+        <h1>Edit page not yet implemented</h1>
+      </div>
+    </BasicLayout>
+  )
+}
```

Comment:
Remove file: There is no use case for a staff edit button. It's only add and delete.